index.html

```diff
@@ -0,0 +1,183 @@
+        </div>
+        <div class="languages">
+            <img class="iconlanguages" src="images/icon-languages.png" alt="">
+            <h2 class="titlelanguages">Languages</h2>
+            <ul class="languagestechs">
+                <li class="Ruby">Ruby</li>
+                <li class="css">css</li>
+                <li class="JavaScript">JavaScript</li>
+                <li class="html">html</li>
+            </ul>
+        </div>
+        <div class="frameworks">
+            <img class="preferredframeworks" src="images/icon-frameworks.png" alt="">
+            <h2 class="titleframeworks">Frameworks</h2>
+            <ul class="techframeworks">
+                <li class="Bootstrap">Bootstrap</li>
+                <li class="Ruby-on-Rails">Ruby on Rails</li>
+                <li class="RSpec">RSpec</li>
+            </ul>
+            <ul class="othertechframeworks">
+                <li class="Capybara">Capybara</li>
+                <li class="Selenium">Selenium</li>
+            </ul>
+        </div>
+        <div class="skills">
+            <img class="myskills" src="images/icon-skills.png" alt="">
+            <h2 class="titleekills">Skills</h2>
+            <ul class="typeofSkills">
+                <li class="Codekit">Codekit</li>
+                <li class="GitHub">GitHub</li>
+                <li class="Codepen">Codepen</li>
+                <li class="Gitlab">Gitlab</li>
+            </ul>
+            <ul class="othertypeofskills">
+                <li class="Terminal">Terminal</li>
+            </ul>
+        </div>
+        <div class="contactinfo">
+            <p class="contactparagraph">I'm always interested in hearing about new projects,
+                 so if you'd like to chat please get in touch.
+            </p>
+            <input class="full-name" type="text" placeholder="Full Name"></br>
+            <input class="email-address" type="email" placeholder="Email address"></br>
+            <input class="enter-text" type="text" placeholder="Enter text here"></br>
+            <button enable="enable" class="get-in-button" type="action">Get in touch</button>
+            <ul class="about-social-container">
+                <li><a class="social-media" href="https://twitter.com/soza_edgard"><i class="fa-brands fa-twitter"></i></a></li>
+                <li><a class="social-media" href="https://www.linkedin.com/in/edgard-soza-sobalvarro/"><i class="fa-brands fa-linkedin-in"></i></a></li>
+                <li><a class="social-media" href="#"><i class="fa-brands fa-m"></i></a></li>
+                <li><a class="social-media" href="https://github.com/edgardsoza"><i class="fa-brands fa-github"></i></a></li>
+                <li><a class="social-media" href="#"><i class="fa-brands fa-angellist"></i></a></li>
+            </ul>
+            <h4 class="my-email">jose.sozasobalvarro@gmail.com</h4>
+            <a><i class="fa-solid fa-dash"></i></a>
+        </div>
```

Comment:
- [ ] Kindly remove this Contact Info it is not required for this milestone and it is also adding to your scroll effect. :+1: 
- [ ] If you want to keep it then I suggest you look for any predefined widths that may be adding to the scroll effect in your project and remove them :wink: